debugger.cpp

```diff
@@ -0,0 +1,1917 @@
+/*

```

Comment:
Git thinks entire `debugger.cpp` / `debugger.h` is new. What happened here? Did you change the newline convention for the entire file? In any case, please revert this as it makes the PR impossible to review.